scalyr_agent/util.py

```diff
@@ -549,11 +559,11 @@ def get_pid_tid():
     # noinspection PyBroadException
     try:
         return "(pid=%s) (tid=%s)" % (
-            str(os.getpid()),
-            str(six.moves._thread.get_ident()),
+            six.text_type(os.getpid()),
+            six.text_type(six.moves._thread.get_ident()),
         )
     except:
-        return "(pid=%s) (tid=Unknown)" % (str(os.getpid()))
+        return "(pid=%s) (tid=Unknown)" % (six.text_type(os.getpid()))
```

Comment:
The return type should be a text string , right?

Reply:
`get_pid_tid` function is mostly used in log messages creation, so, I think yes it should.